src/System.IO/src/System/IO/Stream.cs

```diff
@@ -28,6 +28,36 @@ internal SemaphoreSlim EnsureAsyncActiveSemaphoreInitialized()
             // WaitHandle, we don't need to worry about Disposing it.
             return LazyInitializer.EnsureInitialized(ref _asyncActiveSemaphore, () => new SemaphoreSlim(1, 1));
         }
+        
+        // This method is used by CopyTo, CopyToAsync,
+        // and NullStream's override of the latter.
+        internal void ValidateCopyToArguments(Stream destination, int bufferSize)
```

Comment:
In CoreCLR this method is defined just above the nested `NullStream` class, why not put it in the same place?